clients/shared_library/components/pages/Mailing/components/MailingEditor.tsx

```diff
@@ -23,6 +24,25 @@ export const EmailTemplateEditor = ({
   contentHTMLLabel,
   placeholders,
 }: EmailTemplateEditorProps): JSX.Element => {
+  // Local state to hold warning message
+  const [subjectWarning, setSubjectWarning] = useState('')
+
+  // Regular expression to allow only ASCII characters
+  const asciiOnlyRegex = /^[\x00-\x7F]*$/
```

Comment:
If we restrict input to ASCII characters anyway, we can consider allowing only the printable ones, excluding the first 32 (control) characters.

```suggestion
  const asciiOnlyRegex = /^[\x20-\x7F]*$/
```